examples/cpp/DepthCapture.cpp

```diff
@@ -84,12 +84,23 @@ class VisualizerWithDepthCapture
     }
 };
 
+void PrintHelp() {
+    using namespace open3d;
+
+    PrintOpen3DVersion();
+    // clang-format off
+    utility::LogInfo("Usage:");
+    utility::LogInfo("    > DepthCapture  [filename]");
+    // clang-format on
+    utility::LogInfo("");
+}
+
 int main(int argc, char *argv[]) {
     utility::SetVerbosityLevel(utility::VerbosityLevel::Debug);
-    if (argc < 2) {
-        PrintOpen3DVersion();
-        utility::LogInfo("Usage:");
-        utility::LogInfo("    > DepthCapture  [filename]");
+    if (argc == 1 ||
+        utility::ProgramOptionExistsAny(argc, argv, {"-h", "--help"}) ||
+        argc != 2) {
```

Comment:
argc != 2 || argc == 1 ? Check it out

Reply:
Works as expected, but is a bit over-complicated. Simplified.